www/src/scss/edxorg-theme.scss

```diff
@@ -1,3 +1,5 @@
-@import url("https://cdn.jsdelivr.net/npm/@edx/brand-edx.org@alpha/dist/core.min.css");
-@import url("https://cdn.jsdelivr.net/npm/@edx/brand-edx.org@alpha/dist/light.min.css");
+@import "~paragon-style/scss/core/core";
 @import "base";
+
+@import url("https://cdn.jsdelivr.net/npm/@edx/elm-theme@1.8.1/dist/core.min.css");
+@import url("https://cdn.jsdelivr.net/npm/@edx/elm-theme@1.8.1/dist/light.min.css");
```

Comment:
[inform] At the moment, 2U has no intentions to migrate the existing `@edx/brand-edx.org`. Instead, MFEs will cutover directly to our new tokens-compatible theme `@edx/elm-theme` upon upgrade to a Paragon version that relies on tokens.